applications/FluidDynamicsApplication/python_scripts/navier_stokes_embedded_solver.py

```diff
@@ -231,8 +234,10 @@ def GetDefaultSettings(cls):
             "maximum_iterations": 7,
             "echo_level": 0,
             "time_order": 2,
+            "time_scheme": "bdf2",
```

Comment:
I am wondering that if the user specifies this in the input but the element integrates in time this should be an error?
Or does the element also have different way for the time-integration

Reply:
If the element integrates in time only the BDF2 scheme can be specified (this is why the embedded solver has it in its defaults).
If the user manually specifies another scheme, a meaningful error explaining such incompatibility is thrown.